contrib/completion/bash/docker

```diff
@@ -1504,8 +1523,10 @@ _docker_container_run() {
 			return
 			;;
 		--isolation)
-			__docker_complete_isolation
-			return
+			if [ __docker_daemon_os_is windows ] ; then
```

Comment:
I don't think these `[` and `]` are correct -- this should just be `if __docker_daemon_os_is windows; then`, right?